page/events/introduction-to-custom-events.md

```diff
@@ -78,38 +79,44 @@ Let's make our example a little more interesting. We'll add another room to our
 <div id="master_switch"></div>
 ```
 
-If there are any lights on in the house, we want the master switch to turn all the lights off; otherwise, we want it to turn all lights on. To accomplish this, we'll add two more custom events to the lightbulbs: `turnOn` and `turnOff`. We'll make use of them in the `changeState` custom event, and use some logic to decide which one the master switch should trigger:
+If there are any lights on in the house, we want the master switch to turn all the lights off; otherwise, we want it to turn all lights on. To accomplish this, we'll add two more custom events to the lightbulbs: `light:on` and `light:off`. We'll make use of them in the `light:toggle` custom event, and use some logic to decide which one the master switch should trigger:
 
 ```
-$( ".lightbulb" ).on( "changeState", function( e ) {
+$( ".lightbulb" ).on( "light:toggle", function( event ) {
 	var light = $( this );
-	if ( light.hasClass( "on" ) ) {
-		light.trigger( "turnOff" );
+	if ( light.is( ".on" ) ) {
+		light.trigger( "light:off" );
 	} else {
-		light.trigger( "turnOn" );
+		light.trigger( "light:on" );
 	}
-}).on( "turnOn", function( e ) {
+}).on( "light:on", function( event ) {
 	$( this ).removeClass( "off" ).addClass( "on" );
-}).on( "turnOff", function( e ) {
+}).on( "light:off", function( event ) {
 	$( this ).removeClass( "on" ).addClass( "off" );
 });
 
 $( ".switch, .clapper" ).click(function() {
-	$( this ).parent().find( ".lightbulb" ).trigger( "changeState" );
+	var room = $( this ).closest( ".room" );
+	room.find( ".lightbulb" ).trigger( "light:toggle" );
 });
 
 $( "#master_switch" ).click(function() {
-	if ( $( ".lightbulb.on" ).length ) {
-		$( ".lightbulb" ).trigger( "turnOff" );
+	var lightbulbs = $( ".lightbulb" );
+
+	// check if any lightbulbs are on
```

Comment:
Not sure but comments need a capital letter isn't it? http://contribute.jquery.org/style-guide/js/#comments